docs/CONTRIBUTING.md

```
@@ -1,37 +1,94 @@
+- Keep commit messages clear and meaningful.
+
+Example commit:
+
+```bash
+git commit -m "Fix: Corrected login validation error"
+```
+
+### 6. Push Your Changes
+```
+git push origin feature/your-feature-name
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)